pyccl/halos/halo_model_base.py

```diff
@@ -141,6 +141,7 @@ def _get_fsigma(self, cosmo, sigM, a, lnM):
             float or array_like: :math:`f(\\sigma_M)` function.
         """
 
+    @mass_def_api
```

Comment:
Do we need the same in `get_mass_function`?

Reply:
done